src/App.js

```diff
@@ -1,23 +1,21 @@
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import CountriesList from './components/CountriesList';
+import CountryDetails from './components/CountryDetails';
+import Navbar from './components/Navbar';
+import { Route, Routes } from 'react-router-dom';
 
 function App() {
+  
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Navbar />
+      
+
+      <Routes>
+        <Route path="/" element={ <CountriesList /> } />
+        <Route path="/:countryId" element={ <CountryDetails /> } />
+      </Routes>
     </div>
   );
```

Comment:
1. Because your CountriesList is the main path="/" , you don't have to define a route for it.
2. Use bootstrap to render your countryDetails correctly

```
return (
    <div className="App">
      <Navbar />

      <div className="container">
        <div className="row">
          <CountriesList />
          <Routes>
            <Route path="/:countryId" element={<CountryDetails />} />
          </Routes>
        </div>
      </div>
    </div>
  );
  ```